SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetConnection.cpp

```diff
@@ -115,6 +115,31 @@ void USpatialNetConnection::UpdateActorInterest(AActor* Actor)
 	}
 }
 
+void USpatialNetConnection::NotifyClientHasQuit()
```

Comment:
As this is a client callable function, I'd rename to `ClientNotifyHasQuit`

Reply:
But the name of the field is `client_has_quit`.

Reply:
ClientNotifyClientHasQuit ? We should be following the function naming standard of prefixing function names if they're only intended to be called on client or server.